debian/rules

```diff
@@ -19,7 +19,7 @@ override_dh_auto_build:
 	true
 
 override_dh_installdocs:
-	tox -e manpages
+	tox -i http://pypi.yelpcorp.com -e manpages
```

Comment:
This seems like a step backward. If anything I'd expect the internal pypi to be hard-coded at a higher level so it's used everywhere and not just for this particular manpages invocation.